gdnative-core/src/object/raw.rs

```diff
@@ -126,10 +130,13 @@ impl<T: GodotObject<RefKind = RefCounted>> RawObject<T> {
     /// Increase the reference count of the object.
     #[inline]
     pub fn add_ref(&self) {
-        use crate::ReferenceMethodTable;
-
         let api = crate::private::get_api();
-        let addref_method = ReferenceMethodTable::get(api).reference;
+        let addref_method = unsafe {
+            (api.godot_method_bind_get_method)(
```

Comment:
This is fine for now, but I wonder if we'll want to hand-code a small method table just to avoid these extra FFI calls each time `add_ref`/`unref` is called? Not sure how big the performance cost is here but I guess Godot also does a hashmap lookup on the other side which we might want to avoid?

Reply:
I can add that if you want to. I guess I'd do it with `lazy_static!` or `once-cell`?

Reply:
I guess `once_cell` is better now? We can also enable `parking_lot` since we already depend on it. I guess it's better to just add it to this PR?